bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/internal/MqttBindingConstants.java

```diff
@@ -42,8 +42,9 @@
     public static final String ROLLERSHUTTER = "rollershutter";
     public static final String TRIGGER = "trigger";
 
-    // Color mode
-    public static final String COLOR_MODE_HSB = "hsb";
-    public static final String COLOR_MODE_RGB = "rgb";
-    public static final String COLOR_MODE_XYY = "xyY";
+    public static enum COLOR_MODE {
+        hsb,
```

Comment:
Please use all upper-case for enum members. And as @jochen314 already mentioned: camel-case for the enum-name. Please move the enum to a separate file.